HW_7/task_1.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd 
+import numpy as np 
+import random
+ 
+lst = ['robot'] * 10
+lst += ['human'] * 10
+random.shuffle(lst)
+data = pd.DataFrame({'whoAmI': lst})
+print(data)
+
+data['tmp'] = 1
+data.set_index([data.index, 'whoAmI'], inplace=True)
+data = data.unstack(level=-1, fill_value = 0).astype(int)
+data.columns = data.columns.droplevel()
+data.columns.name = None
+print(data)
```

Comment:
Если я не сбился со счета, то это уже 6 раз, когда мне присылают этот код. Прошу ссылку на источник, но никто не присылает(((

Я тут половину не понимаю.

Reply:
Чем вам более традиционный код не нравится
```suggestion
unique_value = list(set(data.whoAmI))
data[unique_value] = 0
for val in unique_value:
    data.loc[data.whoAmI == val, val] = 1
print(data)
```